package/MDAnalysis/analysis/dssp/pydssp_numpy.py

```diff
@@ -130,6 +131,12 @@ def get_hbond_map(
         input coordinates in either (n, 4, 3) or (n, 5, 3) shape
         (without or with hydrogens). If hydrogens are not present, then
         ideal positions (see :func:_get_hydrogen_atom_positions) are used.
+    donor_mask : np.array
+         Mask out any hydrogens that should not be considered (in particular HN
+         in PRO). If ``None`` then all H will be used (behavior up to 2.9.0).
```

Comment:
These docs should be more specific and state the shape. I just quickly guessed the shape from https://github.com/ShintaroMinami/PyDSSP/blob/e251a43ff8622fe0a555313b1567edce45e789e8/scripts/pydssp#L30
```python
donor_mask = sequence != 'PRO' if args.ignore_proline_donor else None
```

Reply:
Yep, you're right -- and the name `n_atoms` in the function below was wrong, which I guess has caused your confusion. I've added some docs here as well.